lib/data/dataset.js

```diff
@@ -44,10 +44,12 @@ const validateDatasetName = (name) => {
 const getDataset = (xml) => {
   const metaNode = findOne(root('html'), node('head'), node('model'), node('instance'), node(), node('meta'));
   return traverseXml(xml, [
+    findOne(root('html'), node('head'), node('model'))(attr('entities-version')),
     metaNode(findOne(root(), node('entity'))(tree())),
     metaNode(findOne(root(), and(node('entity'), hasAttr('dataset')))(attr('dataset')))
-  ]).then(([ entityTag, datasetName ]) => {
-    // check if <entity> tag exists at all
+  ]).then(([ version, entityTag, datasetName ]) => {
+    if (version.isEmpty() || version.get() !== '2022.1.0')
```

Comment:
We only want to reject if the first two parts don't match! That will let us track changes to the spec that don't require consumer changes. So basically if it doesn't start with 2022.1, reject.